server.js

```diff
@@ -25,15 +39,67 @@ const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
-app.use(bodyParser.json())
+app.use(express.json())
+
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hello from the other side! 🎶')
+})
+
+app.get('/content', async (req, res) => {
+  let data
+  const { country } = req.query
+  const { genre } = req.query
+  const { releaseYear } = req.query
+  const { title } = req.query
+  const { director } = req.query
+  const { cast } = req.query
+
+  try {
+    if (country) {
+      data = await NetflixData.find({ 
+        country: { $regex: country, $options: "i" } 
+      })
+    } else if (genre) {
+      data = await NetflixData.find({ 
+        listed_in: { $regex: genre, $options: "i" }
+      })
+    } else if (releaseYear) {
+      data = await NetflixData.find({ release_year: releaseYear })
+    } else if (title) {
+      data = await NetflixData.find({ 
+        title: { $regex: title, $options: "i" }
+      })
+    } else if (director) {
+      data = await NetflixData.find({ 
+        director: { $regex: director, $options: "i" }
+      })
+    } else if (cast) {
+      data = await NetflixData.find({ 
+        cast: { $regex: cast, $options: "i" }
+      })
+    } else {
+      data = await NetflixData.find()
+    }
```

Comment:
When you give your user many query options to filter your data with, you need to provide logic for every combination of them.